managed/ui/src/components/tables/CreateBackup/CreateBackup.js

```diff
@@ -215,14 +216,20 @@ export default class CreateBackup extends Component {
     }
   };
 
+  /**
+   * This is an onchange event for storage type.
+   * 
+   * @param {string} value Input field value.
+   */
+  backupConfigType = (value) => {
+    this.props.initialValues.storageConfigUUID.value = value;
```

Comment:
Is storageConfigUUID guaranteed to have a `.value`?

Reply:
In `Create Backup` yes in `Restore Backup` no. So I've removed it from `Restore Backup`. I don't understand how it's working for `Restore Backup` because the value will always be undefined in this case. 

Has anyone created a `Restore Backup` And is it working fine?

Reply:
Now I recall that Restore backup doesn't require storageConfig to be set because it is possible to pull it from a S3 bucket. Let's use `YBFormSelect` instead of `YBSelectWithLabel` for this form as well.